backend/src/crash/game_handler.py

```diff
@@ -183,6 +197,17 @@ def is_crashed(self) -> bool:
         else:
             return False  # Game is not crashed, it has not started
 
+    def launch_crash_timer(self):
+        self.crash_event.clear()
+        time.sleep(self.game_duration)
+        self.crash_event.set()
```

Comment:
You could use the sleep_and_go function you defined specifically for this case :D

Reply:
But Choklatier, that is a great observation!
Thank you, I've gone ahead and fixed it.